pandas/_testing.py

```diff
@@ -696,6 +697,10 @@ def assert_index_equal(
         Whether to compare number exactly.
     check_categorical : bool, default True
         Whether to compare internal Categorical exactly.
+    check_order : bool, default True
+        Whether to compare the order of index entries as well as their values.
+        If True, both indexes must contain the same elements, in the same order.
+        If False, both indexes must contain the same elements, but in any order.
```

Comment:
can you add a versionadded tag 1.2 here

Reply:
Oh, that's annoying, sorry.
I removed the versionadded tag when I moved the parameters :-(
I have put it back and rebased again.